server/util/redact/redact.go

```diff
@@ -206,6 +207,41 @@ func redactCmdLine(tokens []string) {
 	stripExplicitCommandLineFromCmdLine(tokens)
 }
 
+func RedactTxt(txt string) string {
+	redacted := stripURLSecrets(txt)
+	redacted = redactRemoteHeaders(redacted)
+	redacted = redactAPIKeys(redacted)
+	return redacted
```

Comment:
uber nit: it feels slightly safer to process the `txt` variable directly, to avoid accidents like `redacted = redactRemoveHeaders(txt)` which redacts the original string instead of the intermediate redacted result

```suggestion
	txt = stripURLSecrets(txt)
	txt = redactRemoteHeaders(txt)
	txt = redactAPIKeys(txt)
	return redacted
```